wrappers/winuser.go

```diff
@@ -231,6 +234,28 @@ func CloseDesktop(desktop syscall.Handle) error {
 	return nil
 }
 
+func GetForegroundWindow() syscall.Handle {
+	r1, _, _ := syscall.Syscall(procGetForegroundWindow.Addr(), 0, 0, 0, 0)
+	return syscall.Handle(r1)
+}
+
+func GetWindowText(hwnd syscall.Handle, buffer *uint16, maxCount int) (int, error) {
+	r1, _, e1 := syscall.Syscall(
+		procGetWindowText.Addr(),
+		3,
+		uintptr(hwnd), uintptr(unsafe.Pointer(buffer)),
+		uintptr(maxCount))
+	if e1 != ERROR_SUCCESS {
+		return int(r1), e1
+	}
+	return int(r1), nil
+}
+
+func GetWindowThreadProcessId(hwnd syscall.Handle, processID *uint32) syscall.Handle {
+	r1, _, _ := syscall.Syscall(procGetWindowThreadProcessId.Addr(), 2, uintptr(hwnd), uintptr(unsafe.Pointer(processID)), 0)
+	return syscall.Handle(r1)
```

Comment:
Even though the documentation from Microsoft doesn't say anything about error handling for this function, if you call it with an invalid window handle, it will return 0 and set `GetLastError` to `ERROR_INVALID_WINDOW_HANDLE`.  We should change the return type to `(syscall.Handle, error)` so that users of the library can detect this error case if they wish to do so.